src/main/java/seedu/cafectrl/storage/CorruptedDataException.java

```diff
@@ -0,0 +1,4 @@
+package seedu.cafectrl.storage;
+
+public class CorruptedDataException extends Exception{
```

Comment:
Maybe add a whitespace here? :)

```suggestion
public class CorruptedDataException extends Exception {
```